### PR TITLE
Few easy performance wins

### PR DESCRIPTION
**Pull request by egonelbre**
Overall win for a ogg file I had `93.2ms -> 68.9ms`.

### CODE REVIEW

File: internal/silk/decoder.go
Comment (egonelbre) [inline internal/silk/decoder.go:1405]: I realized I made a mistake here, I'll fix a bit later.

File: internal/rangecoding/decoder.go
Comment (Sean-Der) [inline internal/rangecoding/decoder.go:192]: What do you think of making `1>>23` a constant? When I was reading ffmpeg I didn't understand where this value came from at first. I love the optimization :)

```
// Value equal to 2^23
const twoToTwentyThird = 1 << 23
```

File: internal/silk/silk.go
Comment (Sean-Der) [inline internal/silk/silk.go:79]: Where does the term `unit` come from? All for the change, but `clampUnit` would confuse me at first.

I think something more verbose would be one `clampNegativeOneToOne` even.

**Comment by Sean-Der** [comment]
Amazing changes @egonelbre ! Love it, so excited to work together on this

* Can I add you to the Pion org?
* Want to land these now as small commits?

File: internal/rangecoding/decoder.go
Comment (egonelbre) [reply internal/rangecoding/decoder.go:192]: I would use:
```
// maxRangeSize is ... It's equal to 2^23.
const maxRangeSize = 1 << 23
```

File: internal/silk/silk.go
Comment (egonelbre) [reply internal/silk/silk.go:79]: Ah, `unit` is one of the names I've been using myself for denoting -1 to 1 range. I cannot recall where I picked it up -- it could've been something related to graphics programming -- i.e. clamp to a unit vector.

Maybe `clamp11` or just `clamp1` would be easier to read? But, since it's not too commonly used here, `clampNegativeOneToOne` would be fine as well.

File: internal/rangecoding/decoder.go
Comment (Sean-Der) [reply internal/rangecoding/decoder.go:192]: @egonelbre LGTM! If you want to push commit directly to master (added you to Pion org) or open a PR with that I can merge right now.

Comment (egonelbre) [reply internal/rangecoding/decoder.go:192]: I'm currently updating the changes and update the PR, as it's easier at this point. It'll take me 5-10 min to fix the issues.

Comment (egonelbre) [reply internal/rangecoding/decoder.go]: done

**Comment by egonelbre** [comment]
@Sean-Der sure, you can add me to the org; but I still like the review process as it allows to catch some mistakes.

File: internal/silk/decoder.go
Comment (Sean-Der) [inline internal/silk/decoder.go:1396]: Can you move these up with `outVal` or move `outVal` into this scope?

File: internal/silk/decoder.go
Comment (Sean-Der) [reply internal/silk/decoder.go:1406]: @egonelbre Is this fixed/ok to merge?

**Comment by Sean-Der** [comment]
@egonelbre great! One last question/point and I am happy to merge :)

**Review by Sean-Der — APPROVED** [review]

**Comment by Sean-Der** [comment]
@egonelbre go ahead and merge w/e you are happy with the PR :)

File: internal/silk/decoder.go
Comment (egonelbre) [reply internal/silk/decoder.go:1405]: Yes, it's fixed now.

**Pull request merged**
